x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -567,6 +585,14 @@ export class TaskRunner<
       executionStatus.lastExecutionDate = new Date(event.event.start);
     }
 
+    if (apm.currentTransaction) {
+      if (executionStatus.status === 'ok') {
```

Comment:
The valid status values for this field are here:

https://github.com/elastic/kibana/blob/7c8b6b6f9e6ca9738c619b66a8c6a6bf50b3d77d/x-pack/plugins/alerting/common/alert.ts#L24-L25

`pending` means it hasn't run yet, so should never go through this path.

`unknown` indicates some unexpected issue, so should probably be categorized as 'failure'.

`active` is a flavor of `ok` - the difference between the two is `active` indicates there are active alert instances, `ok` indicates there are not (the last run did not schedule any alert instances).  So `active` should also be listed as `success`.

Reply:
Handled both suggested cases 👍